include/flang/Optimizer/Dialect/FIROps.h

```diff
@@ -68,6 +67,8 @@ mlir::ParseResult parseSelector(mlir::OpAsmParser &parser,
                                 mlir::OpAsmParser::OperandType &selector,
                                 mlir::Type &type);
 
+// Sources generated with tablegen require mlir namespace to be accessible.
+using namespace mlir;
```

Comment:
This seems a little strange to be near the bottom of the file to me. Is this a maintenance mistake?

Reply:
No, that was just to make it only effective on the generated FIROps.h.inc and also so that the using directive from this header does not "spill" from this header in every included file.

Reply:
I had to move this back because of compiler errors. I didn't spend any time on trying to fix those errors.